src/package.rb

```diff
@@ -24,8 +24,7 @@
     'EvEx.rb',
     'Commands.rb',
     'Incubator.rb',
-    'Tools.rb',
-    'Doc.rb'
```

Comment:
Are you sure ? If I remember, the "eval ingame" use the data of Doc.rb, no?

Reply:
checked and ... he does not use it. :v

Reply:
I re-check

Reply:
I confirm. :)